code/modules/client/preference_setup/loadout/lists/utility.dm

```diff
@@ -64,12 +64,31 @@
 modular computers
 ****************/
 
+/datum/gear/utility/cheaptablet
+	display_name = "tablet computer, cheap"
+	path = /obj/item/modular_computer/tablet/preset/custom_loadout/cheap
+	cost = 3
+
+/datum/gear/utility/normaltablet
+	display_name = "tablet computer, advanced"
+	path = /obj/item/modular_computer/tablet/preset/custom_loadout/advanced
+	cost = 4
+
+/datum/gear/utility/customtablet
+	display_name = "tablet computer, custom"
+	path = /obj/item/modular_computer/tablet
+	cost = 4
+
+/datum/gear/utility/customtablet/New()
+	..()
+	gear_tweaks += new /datum/gear_tweak/tablet()
+
 /datum/gear/utility/cheaplaptop
 	display_name = "laptop computer, cheap"
 	path = /obj/item/modular_computer/laptop/preset/custom_loadout/cheap
-	cost = 5
```

Comment:
With the cost being equal, why would you go for laptop vs tablet, or vice versa?

Reply:
Portability. Size. Pocketing. Convenience. I never got laptops, but laptops can run certain programs that Tablets can't.